Invoking command in '/home/aysenur/path_planning_ws/src/maze_bot': PYTHONPATH=/home/aysenur/path_planning_ws/build/maze_bot/prefix_override:/home/aysenur/path_planning_ws/install/maze_bot/lib/python3.8/site-packages:${PYTHONPATH} /usr/bin/python3 setup.py egg_info --egg-base ../../build/maze_bot build --build-base /home/aysenur/path_planning_ws/build/maze_bot/build install --record /home/aysenur/path_planning_ws/build/maze_bot/install.log --single-version-externally-managed
Invoked command in '/home/aysenur/path_planning_ws/src/maze_bot' returned '0': PYTHONPATH=/home/aysenur/path_planning_ws/build/maze_bot/prefix_override:/home/aysenur/path_planning_ws/install/maze_bot/lib/python3.8/site-packages:${PYTHONPATH} /usr/bin/python3 setup.py egg_info --egg-base ../../build/maze_bot build --build-base /home/aysenur/path_planning_ws/build/maze_bot/build install --record /home/aysenur/path_planning_ws/build/maze_bot/install.log --single-version-externally-managed
